cmd/gitserver/server/patch.go

```diff
@@ -62,6 +62,7 @@ func (s *Server) createCommitFromPatch(ctx context.Context, req protocol.CreateC
 	var resp protocol.CreateCommitFromPatchResponse
 
 	repo := string(protocol.NormalizeRepo(req.Repo))
+	repoDir := filepath.Join(s.ReposDir, repo)
 	repoGitDir := filepath.Join(s.ReposDir, repo, ".git")
```

Comment:
makes their relation a bit clearer imo
```suggestion
	repoGitDir := filepath.Join(repoDir, ".git")
```